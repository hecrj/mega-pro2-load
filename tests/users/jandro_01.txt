6

100
200
500
600
1000
300

9

7 9 6 2 0 0 4 0 0 0 1 3 0 5 0 0 8 0 0

10
1 6
20
2 6 2
30
1 6
40
1 5
50
2 5 2
60
1 5
70
3 2 3 1
80
2 3 6
90
1 3

-1 4 0

-4 9
1 4 
0

-4 1
1 4
0

-1 4 20

-1 4 21

-2 26

-2 27

-3 28

-4 1
0
0

-3 29

-5 0 20

-5 0 21

-5 500 600

-1 6 35

-1 5 50

-6



