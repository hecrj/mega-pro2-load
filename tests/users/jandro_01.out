Peticion procesada y servidores
0 0
Peticion procesada y servidores
1 7
9 
Peticion procesada y servidores
2 60
1 
Peticiones pendientes
1 4 20 27
2 4 21 81
Peticiones pendientes
2 4 21 81
Servidores ocupados
1 2 53
Servidores ocupados
1 2 52
Pelicula mas solicitada
4 1
Pelicula mas solicitada
4 2
Pelicula mas solicitada
0 0
Peticion procesada y servidores
3 4
8 
Peticion procesada y servidores
4 10
4 6 
